app/Models/News.php

```diff
@@ -25,6 +25,6 @@ public function media()
 
     public function categoryNews()
     {
-        return $this->belongsTo('App\Models\CategoryNews');
+        return $this->belongsTo('App\Models\CategoryNews', 'cate_id', 'id');
```

Comment:
như này cho nó gọn em
return $this->belongsTo(News::class, 'cate_id', 'id');

Reply:
Vâng ạ